airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/MssqlSourceDatatypeTest.java

```diff
@@ -225,7 +225,7 @@ protected void initTests() {
             .sourceType("time")
             .airbyteType(JsonSchemaType.STRING)
             .addInsertValues("null", "'13:00:01'", "'13:00:04Z'", "'13:00:04.123456Z'")
```

Comment:
It looks like we expect to lose "Z" - is this the correct behavior?

Reply:
I just aligned it to what we already use in other connectors (re-used existing time format). Should I add an additional format to add it back? Thanks